src/components/mobileservices/validator/rules/SameValueOfRule.test.js

```diff
@@ -0,0 +1,69 @@
+import { SameValueOfRule } from './SameValueOfRule';
+
+describe('SameValueOfRule Validation', () => {
+  it('target value is a undefined', () => {
```

Comment:
I think it would be better if the `it` text describes what should happen.

```suggestion
it('should be valid when target value is undefined', () => {
```

This goes for every `it` function in the test file.